LandSandBoat/server
# Pathos effects changes (salvage and nyzul usage)

**_I affirm:_**

## What does this pull request do?

1. Adds a new effect flag that will only allow wearing of the debuff to only happen from zoning into a non instanced zone.
with this people can D/C in salvage and nyzul and the buff will never wear off, only by leaving the instance will it, since every instanced zone is connected to a non instanced zone.

2. Applies pathos a player may have on to all pets when called upon through core.
3. Removes all pathos from pets as they are removed from them either by cell usage-salvage or by floor progress-nyzul
4. Adds timing to pathos inflicted on entering salvage so the effect cant be removed from logging out. 
5. Adds powers to Omerta to differentiate between white magic, black magic, blue magic, ect - nyzul usage 
6. Adds powers to Impairment to differentiate between weaponskill and job ability usage - nyzule usage

## Steps to test these changes

!instance 7400
!changejob pup, smn, drg, or bst
!deleffect 262 to call a smn
!deleffect 261 for all other jobs

pup is easiest since you see all stats.
add cells and use and see that cells effect pets when applied to master
logout and in, dismiss pet and recall should all be how it was.
use fireflies to escape and when out of instance all effects were removed
